components/service-binding-usage-controller/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.10-alpine3.10 as builder
+FROM golang:1.13.10-alpine3.10 as builder
```

Comment:
Is there any reason we use golang 1.13.10 version instead for example 1.14.2 like in `buildpack-golang-toolbox` version `v20200423-1d9d6590`?